Doc/c-api/intro.rst

```diff
@@ -832,6 +832,13 @@ It is also implied by the presence of the
 not-Python-specific :c:macro:`!_DEBUG` macro.  When :c:macro:`!Py_DEBUG` is enabled
 in the Unix build, compiler optimization is disabled.
 
+.. note::
+   On Windows, the :c:macro:`!Py_DEBUG` macro is not defined by default.
+
+   The :option:`--with-pydebug` option is not available on Windows. Instead, the
+   not-Python-specific :c:macro:`!_DEBUG` macro can be defined by the compiler
+   options and that will enable the :c:macro:`!Py_DEBUG` macro.
+
```

Comment:
There's no need for this to be a note, we can just mix in the Windows info with the existing docs (e.g. passing `-d` to `PCbuild.bat`).

The wording is a bit weird, since on Windows you choose a debug build and get `Py_DEBUG`, whereas the text makes it sound like on Unix you choose `Py_DEBUG` and so get a debug build (which also seems not quite right).  The whole section might benefit from improvement.